CourseManager/Courses/Course 4 UI and Animations in SwiftUI/Views/Components/ForecastCard.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  ForcastCard.swift
+//  CourseManager
+//
+//  Created by Summer Crow on 2022-11-10.
+//
+
+import SwiftUI
+
+struct ForecastCard: View {
+    var forecast: Forecast
+    var forecastPeriod: ForecastPeriod
+    var isActive: Bool {
+        if forecastPeriod == ForecastPeriod.hourly {
```

Comment:
It is better to use `switch` statement here

Reply:
Oh yes ofcourse. Updated to this: 
   var isActive: Bool {
        switch forecastPeriod {
        case .hourly:
            let isThisHour = Calendar.current.isDate(.now, equalTo: forecast.date, toGranularity: .hour)
            return isThisHour
        case .daily:
            let isToday = Calendar.current.isDate(.now, equalTo: forecast.date, toGranularity: .day)
            return isToday
        }
    }